unlock-protocol-com/src/components/pages/Home/sections/Connect.tsx

```diff
@@ -6,9 +6,9 @@ import { BulletPointIcon } from '../../../icons'
 import Autoplay from 'embla-carousel-autoplay'
 
 const UNLOCK_BENEFITS = [
-  'Create memberships and sell access NFTs in minutes',
+  'Create memberships contracts and sell access NFTs in minutes',
```

Comment:
@julien51 should be "membership contracts" (no "s" on membership)

Reply:
```suggestion
  'Create membership contracts and sell access NFTs in minutes',
```